ArmoniK.Extension.CSharp.Client/Handlers/SessionHandler.cs

```diff
@@ -21,63 +21,100 @@
 
 namespace ArmoniK.Extension.CSharp.Client.Handlers;
 
+/// <summary>
+///   Handles session management operations for an ArmoniK client, providing methods to control the lifecycle of a
+///   session.
+/// </summary>
 public class SessionHandler
 {
-  public readonly  ArmoniKClient ArmoniKClient;
+  private readonly ArmoniKClient armoniKClient_;
```

Comment:
I think we can expose underlying clients. It should even be part of the Interface